core/main.tf

```diff
@@ -71,3 +73,20 @@ module "ecs" {
   secrets_variables = module.secrets_manager.secrets_variables
   secret_arns       = module.secrets_manager.secret_arns
 }
+
+module "rds" {
+  source = "../modules/rds"
+
+  vpc_security_group_ids = module.security_group.rds_security_group_ids
+  vpc_id                 = module.vpc.vpc_id
+  subnet_ids             = module.vpc.private_subnet_ids
+
+  instance_type = var.rds_instance_type
+  database_name = var.environment
+  username      = var.rds_username
+  password      = var.rds_password
+  port          = local.rds_port
+
+  autoscaling_min_capacity = var.rds_autoscaling_min_capacity
+  autoscaling_max_capacity = var.rds_autoscaling_max_capacity
```

Comment:
The following configurations should be controlled by local values instead of variables
- instance_type
- port
- autoscaling_min_capacity
- autoscaling_max_capacity

Reply:
Thanks for raising it, I agree with your comment that we should define it as `locals` instead of `input` variables, but for `port`, I prefer to keep it as an input variable instead since we also need to use it in the `security group`. (which was defined as a `local` variable in the `core` folder).

Reply:
Seems like I misunderstood your comment, I updated it in d93e502 ✅